lib/tasks/account.rake

```diff
@@ -6,5 +6,10 @@ namespace :hyku do
     task :create, [:name] => [:environment] do |_t, args|
       CreateAccount.new(Account.new(name: args[:name])).save
     end
+    desc 'Update the frontend url of an account'
+    task :frontend_url, [:tenant, :frontend_url] => [:environment] do |_t, args|
+      account = Account.find_by(tenant: args[:tenant])
+      UpdateAccountFrontendUrl.new(account, args[:frontend_url]).save
```

Comment:
I think it would be useful to have some feedback if this operation succeeded or not.

Reply:
That's a good point. I will make that happen